.gitignore

```diff
@@ -73,15 +73,15 @@ node_modules
 package-lock.json
 
 ## Rustdoc GUI tests
-src/test/rustdoc-gui/src/**.lock
+tests/rustdoc-gui/src/**.lock
 
 # Before adding new lines, see the comment at the top.
 /.litani_cache_dir
 /.ninja_deps
 /.ninja_log
-/src/test/bookrunner
+/docs/bookrunner
```

Comment:
docs? not tests?

Reply:
I messed up this one while trying to understand how build docs script publishes the book runner result. Thanks for catching it up.